Dockerfile-checklicence

```diff
@@ -46,7 +46,8 @@ ENV RAT_VERSION="${RAT_VERSION}" \
     RAT_URL="http://repo1.maven.org/maven2/org/apache/rat/apache-rat/${RAT_VERSION}/apache-rat-${RAT_VERSION}.jar"
```

Comment:
This has to be `https://`. I know that this isn't part of the PR, but we need to fix this.

Reply:
Ah Right! fixed it as well ! Good catch @Fokko :)